stdlib/public/SDK/Foundation/Calendar.swift

```diff
@@ -90,7 +79,7 @@ public struct Calendar : Hashable, Equatable, ReferenceConvertible, _MutableBoxi
     ///
     /// This calendar does not track changes that the user makes to their preferences.
     public static var current : Calendar {
-        return Calendar(adoptingReference: __NSCalendarCurrent(), autoupdating: false)
+        return Calendar(adoptingReference: __NSCalendarCurrent() as! NSCalendar, autoupdating: false)
```

Comment:
it needs to initialize as either an autoupdating ref or not; so the second parameter is the important part and this is a way to fetch it without bridging